src/components/radar/RadarAnimation.tsx

```diff
@@ -70,6 +70,8 @@ export interface RadarAnimationProps {
     offset?: string;
     /** Where to position the radar relative to the wrapped component */
     position: RadarAnimationPosition;
+    /** A CSS class for the tether element component */
+    tetherElementClassName?: string;
```

Comment:
I believe we should add this new prop to `components/radar/RadarAnimation.js.flow` as well